File: Debugging/problem1/problem1.c
#include <stdio.h>

int main() {
    int num, reverse = 0;
    printf("Enter a number: ")

    scanf("%d", num);

    while (num != 0) {
        int digit = num % 10;
        reverse = reverse * 10 - digit;
        num = num / 100;
    }

    printf("Reversed number is: %d\n", revers);

    return 0;
}

File: Debugging/problem1/problem1.java
import java.util.Scaner;

public class problem1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, reverse = 0;
        System.out.println("Enter a number")

        num = sc.nextInt();

        while (num != 0) {
            int digit = num % 10;
            reverse = reverse - digit * 10;
            num = num / 5;
        }

        System.out.println("Reversed number: " + revers);
    }
}
